### DISCUSSION TITLE
win11 22h2 任务栏显示错位

### DISCUSSION THREAD

**Discussion by yanghaoynu** [zhongyang219/TrafficMonitor#1603] [General]
感谢作者，工具非常实用。
版本1.84.1x64，已经尝试过不缩放以及其他缩放比例，都一样的效果，请作者抽空看看是否可以处理。
![Snipaste_2023-05-02_06-50-05](https://user-images.githubusercontent.com/34472989/235544990-42c2430c-b047-450d-b899-b1a727583fad.png)

**Comment by alinix** [comment]
我也遇到这个问题，预发行版本作者已经解决，等作者更新吧

**Reply by yanghaoynu** [reply]
目前用raimeter插件替代，看作者啥时候发布吧。
![Snipaste_2023-05-26_11-41-19](https://github.com/zhongyang219/TrafficMonitor/assets/34472989/f24cdb7d-96ea-43b7-b7d7-8265f7b39876)

**Comment by alinix** [comment]
![20230526163536](https://github.com/zhongyang219/TrafficMonitor/assets/28524286/c491f8d3-6b8e-40c6-a471-506cff87a8fc)
一样的壁纸
